modules/embeddingsstore/src/main/java/com/theagilemonkeys/ellmental/embeddingsstore/pinecone/PineconeEmbeddingsStore.java

```diff
@@ -22,6 +22,8 @@ public class PineconeEmbeddingsStore extends EmbeddingsStore {
     private String namespace;
     // This attribute has no modifier to be package-private, so it can be mocked in tests.
     // This field will be accessible within this package, but not outside of it.
+    // In the future, we can see if MockWebServer from OkHttpClient works to simplify this:
+    // https://github.com/square/okhttp/blob/master/mockwebserver-junit5/README.md
```

Comment:
Let's add the TODO string so it's detected by the IDE
```suggestion
    // TODO: Check if MockWebServer from OkHttpClient can simplify this:
    // https://github.com/square/okhttp/blob/master/mockwebserver-junit5/README.md
```